scripts/rollup/bundles.js

```diff
@@ -36,242 +36,88 @@ const RENDERER_UTILS = moduleTypes.RENDERER_UTILS;
 
   /******* React DOM Server *******/
   {
-    babelOpts: babelOptsReact,
+    label: 'dom-server-browser',
     bundleTypes: [UMD_DEV, UMD_PROD, NODE_DEV, NODE_PROD, FB_DEV, FB_PROD],
-    config: {
-      destDir: 'build/',
-      globals: {
-        react: 'React',
-      },
-      moduleName: 'ReactDOMServer',
-      sourceMap: false,
-    },
-    entry: 'packages/react-dom/server.browser.js',
-    externals: ['prop-types', 'prop-types/checkPropTypes'],
-    fbEntry: 'packages/react-dom/server.browser.js',
-    hasteName: 'ReactDOMServer',
     moduleType: RENDERER,
-    label: 'dom-server-browser',
-    manglePropertiesOnProd: false,
-    name: 'react-dom/server.browser',
-    paths: [
-      'packages/events/**/*.js',
-      'packages/react-dom/*.js',
-      'packages/react-dom/src/server/**/*.js',
-      'packages/react-dom/src/shared/**/*.js',
-      'packages/shared/**/*.js',
-    ],
+    entry: 'react-dom/server.browser',
+    global: 'ReactDOMServer',
+    externals: ['react'],
   },
 
   {
-    babelOpts: babelOptsReact,
+    label: 'dom-server-node',
     bundleTypes: [NODE_DEV, NODE_PROD],
-    config: {
-      destDir: 'build/',
-      globals: {
-        react: 'React',
-      },
-      moduleName: 'ReactDOMNodeStream',
-      sourceMap: false,
-    },
-    entry: 'packages/react-dom/server.js',
-    externals: ['prop-types', 'prop-types/checkPropTypes', 'stream'],
     moduleType: RENDERER,
-    label: 'dom-server-server-node',
-    manglePropertiesOnProd: false,
-    name: 'react-dom/server.node',
-    paths: [
-      'packages/events/**/*.js',
-      'packages/react-dom/*.js',
-      'packages/react-dom/src/server/**/*.js',
-      'packages/react-dom/src/shared/**/*.js',
-      'packages/shared/**/*.js',
-    ],
+    entry: 'react-dom/server',
+    output: 'react-dom/server.node',
+    externals: ['react', 'stream'],
   },
 
   /******* React ART *******/
   {
-    babelOpts: babelOptsReactART,
-    // TODO: we merge react-art repo into this repo so the NODE_DEV and NODE_PROD
-    // builds sync up to the building of the package directories
+    label: 'art-fiber',
     bundleTypes: [UMD_DEV, UMD_PROD, NODE_DEV, NODE_PROD, FB_DEV, FB_PROD],
-    config: {
-      destDir: 'build/',
-      globals: {
-        react: 'React',
-      },
-      moduleName: 'ReactART',
-      sourceMap: false,
-    },
-    entry: 'packages/react-art/index.js',
-    externals: [
-      'art/modes/current',
-      'art/modes/fast-noSideEffects',
-      'art/core/transform',
-      'prop-types/checkPropTypes',
-      'react-dom',
-    ],
-    fbEntry: 'packages/react-art/index.js',
-    hasteName: 'ReactART',
     moduleType: RENDERER,
-    label: 'art-fiber',
-    manglePropertiesOnProd: false,
-    name: 'react-art',
-    paths: [
-      'packages/react-art/**/*.js',
-      'packages/react-reconciler/**/*.js',
-      'packages/shared/**/*.js',
-    ],
+    entry: 'react-art',
+    global: 'ReactART',
+    externals: ['react'],
+    babel: opts =>
```

Comment:
nit: This could have a more meaningful name? Maybe like `babelOptionsHook`? I dunno. Names are hard.

Reply:
I'll probably do another pass renaming them in this PR. I oscillated between long and short names here and used short ones. But maybe I was just tired of complex config and tried to simplify the perception by using shorter words. I don't care strongly either way and will think about them again.